Rakefile

```diff
@@ -21,14 +23,17 @@ require 'bundler_api/gem_helper'
 $stdout.sync = true
 Thread.abort_on_exception = true
 
+RuboCop::RakeTask.new
+
 begin
   require 'rspec/core/rake_task'
 
   desc "Run specs"
   RSpec::Core::RakeTask.new(:spec) do |t|
     t.rspec_opts = %w(-fs --color)
-    #t.ruby_opts  = %w(-w)
+    Rake::Task['rubocop'].invoke
```

Comment:
Augh, I'm terrible. It has to be `task :spec => :rubocop`. That tells rake that the spec task can't run until the rubocop task has run.